verifier/src/tcb.rs

```diff
@@ -672,20 +674,42 @@ mod tests {
         let time = "2023-08-11T19:56:44Z"
             .parse::<DateTime>()
             .expect("Failed to parse time");
+        let time = Some(time);
 
         assert_matches!(
             signed_tcb_info.verify(Some(&key), time),
             Err(Error::TcbInfoExpired)
         );
     }
 
+    #[test]
+    fn passes_before_issue_date_ignoring_time() {
```

Comment:
don't think "before" matters since time is None
```suggestion
    fn passes_ignoring_time() {
```